ansible/roles/nim-waku/templates/docker-compose.yml.j2

```diff
@@ -41,6 +45,12 @@ services:
       --persist-messages=true
       --lightpush=true
       --swap={{ nim_waku_swap_protocol_enabled | lower }}
+  {% if nim_waku_websocket_enabled %}
```

Comment:
I'm not exactly sure why we need this global config enable/disable, though it might be an infra config requirement?

Reply:
Ah, but I notice this was discussed in a previous comment.

Reply:
Yes, because we don't want to accidentally roll this out to prod before the image is updated and break the fleet.